From https://github.com/unispeech/unimrcp/issues/147#issuecomment-120516196

Below is a one-line change which fixes the problem. The problem was in the Sofia-SIP
library which deleted a data storage key which was not created. That eventually made
Asterisk crash because of memory corruption issues. The fix will be available in the
upcoming release of the dependencies.

diff --git a/libsofia-sip-ua/su/su_uniqueid.c b/libsofia-sip-ua/su/su_uniqueid.c
index 59833cc..eaa8813 100644
--- a/libsofia-sip-ua/su/su_uniqueid.c
+++ b/libsofia-sip-ua/su/su_uniqueid.c
@@ -99,7 +99,7 @@ union state {
 #endif

 static pthread_once_t once = PTHREAD_ONCE_INIT;
-static int done_once = 1;
+static int done_once = 0;
 static pthread_key_t state_key;

